README.md

```diff
@@ -430,14 +430,16 @@ In order to have the proxy running, there are several basic pieces of informatio
     port: 10026
 },
 ```
-* `config.access`: connection information to the selected Access Control PDP API. Includes a `disable` flag, to allow the proxy to work in authentication-only mode. E.g.:
+* `config.access`: connection information to the selected Access Control PDP API. Includes a `disable` flag, to allow the proxy to work in authentication-only mode and a flag to activate accounting access in file. E.g.:
```

Comment:
It is great to include new config items in this section but I think we also need some brief general documentation about the new feature. Maybe a small section (one or two paragraph) explaining how it works and the format used by the access log file.